docs: adding splash screen docs (#3703)
unoplatform/uno: doc/articles/uno-development/splash-screen.md

# Splash Screens

> Navigate to `YourProject.Shared > Assets` and add all [scales](#table-of-scales) of your spash screen image.

Reviewer: ```suggestion
This article covers how to add a splash screen to your application.

As a first step, navigate to `YourProject.Shared > Assets` and add all [scales](#table-of-scales) of your splash screen image.
```

## UWP

In your UWP project head, navigate to `Package.appxmaifest > Visual Assets > Splash Screen` and add your images.

 ![uwp-splash-screen](assets/uwp-splash-screen.JPG)

## Android 

1. In the Android project head, navigate to `Resources/values/Styles.xml`

2. Add an `android:windowBackground` `item` to the `AppTheme` style.  

> ```xaml

Reviewer: ```suggestion
```xml
```

<item name=�android:windowBackground�>@drawable/splash</item>
```

3. Navigate to `Resources/drawable`

4. Create a `splash.xml` drawable  

> ``` xaml

Reviewer: ```suggestion
``` xml
```

Author: In an earlier PR, Jerome recommended I use the xaml tag which is what I have been using. I am happy to change it, I just want to make sure I stay consistent

Reviewer: It's good to use the `xaml` tag whenever it applies, but since this is a native Android resource it's not using Xaml, so plain `xml` makes more sense 😉

<?xml version=�1.0� encoding=�utf-8�?>
<layer-list xmlns:android=�http://schemas.android.com/apk/res/android�>
<item>
<!� background color �>
<color android:color=�#101010�/>
</item>
<item>
<!� splash image �>
<bitmap
android:src=�@drawable/splashscreen�
android:tileMode=�disabled�
android:gravity=�center� />
</item>
</layer-list>
```

## iOS 

1. Right-click and select `Add > New item`

2. Create a new `StoryBoard` named `SplashScreen.storyboard` 

3. In the `Toolbox` window, drag and drop a `View Controller` and then an `ImageView` inside the `View Controller`. 

![viewcontroller-imageview](assets/viewcontroller-imageview.png)

4. To have an image fill the screen, set your constraints as below

![ios-constraintes](assets/ios-constraints.png)

5. Set the `Content Mode` to `Aspect Fit` 

![ios-content-fit](assets/ios-content-fit.png)

6. In the `Properties > Storyboard Document` window, select the `Can be Launch Screen` checkbox.

![can-be-launch](assets/can-be-launch.png)

7. Close the designer and open the `.storyboard` file.

8. Add your image path to the `Image View`

> ``` xaml

Reviewer: ```suggestion
``` xml
```

<imageView � image=�Assets/SplashScreen�>
```

9. Navigate to `info.plist > Visual Assets > Launch Images` and update the `Launch Screen` value to `SplashScreen`.

## WebAssembly

1. In the Wasm project head, navigate to `WasmScripts/AppManifest.js` 

2. Add your splash screen image

> ``` xaml

Reviewer: ```suggestion
``` javascript
```

var UnoAppManifest = {
splashScreenImage: �Assets/SplashScreen.scale-200.png�,
splashScreenColor: �#101010�,
displayName: �Uno.SplashScreenEverywhere�
}
```

> Note: Currently, you need to set an explicit scale of the splash screen image.

Reviewer: Is this referring to the WASM instructions above or the table below?

@ebariche do you think this still applies after your changes?

Reviewer 2: Yes it still applies (the file is loaded by HTML explicitly)
